packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 5.0.3 (Unreleased)
+
+### Polish
```

Comment:
"Polish" to me seems like it should be used exclusively for non-impacting refactorings. This seems more an enhancement.